modules/web/src/app/settings/admin/seed-configurations/template.html

```diff
@@ -0,0 +1,128 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card>
+  <mat-card-header>
+    <mat-card-title fxFlex
+                    fxLayout="row"
+                    fxLayoutAlign=" center">
+      <div fxFlex>Seed Configurations</div>
+    </mat-card-title>
+  </mat-card-header>
+
+  <table class="km-table"
+         mat-table
+         matSort
+         [dataSource]="dataSource">
+
+    <ng-container [matColumnDef]="Column.Phase">
+      <th mat-header-cell
+          *matHeaderCellDef
+          class="km-header-cell p-5"></th>
+      <td mat-cell
+          *matCellDef="let element">
+        <i [matTooltip]="element.status"
+           [ngClass]="getStatusColor(element)"></i>
+      </td>
+    </ng-container>
+
+
+    <ng-container [matColumnDef]="Column.Name">
+      <th mat-header-cell
+          *matHeaderCellDef
+          class="km-header-cell p-15"
+          mat-sort-header>Name
+      </th>
+      <td mat-cell
+          *matCellDef="let element">{{element[Column.Name]}}</td>
+    </ng-container>
+
+
+    <ng-container [matColumnDef]="Column.Providers">
+      <th mat-header-cell
+          *matHeaderCellDef
+          class="km-header-cell p-30">Providers
+      </th>
+      <td mat-cell
+          *matCellDef="let element"
+          fxLayout="row"
+          fxLayoutAlign=" center"
+          class="providers-column">
+        <ng-container *ngFor="let provider of getDisplayedProviders(element)">
+          <span class="km-provider-logo km-provider-logo-{{provider}}">
+          </span>
+        </ng-container>
+        <span *ngIf="element[Column.Providers]?.length > displayedProviders"
+              class="km-text km-pointer">
+          +{{element[Column.Providers]?.length - displayedProviders}}
```

Comment:
`Column.Providers = providers` and element is for `SeedOverviewDatasource` interface right?
and `SeedOverviewDatasource` don't have key `providers` i think this is not working right ?or i miss somthing
i think the `[*ngIf]` always return false

Reply:
SeedOverviewDatasource does have providers.The payload is feeded with providers by "...spread" operator  but i understand i can add providers in Interface for clarity.

_i think this is not working right ?or i miss somthing i think the [*ngIf] always return false_ 

You missed something but UI is working and to make clear i can add property as i have explained above.

https://github.com/kubermatic/dashboard/pull/5285/files/02f3426f918870e277feeeb83f8b0d38a58427f7#diff-d148e501bad6de9a4ca40052aabe269db382fdd508051b6fca4a8800d088ab14R58

Reply:
`element[Column.Providers]?.length` is `undefined` because `element[Column.Providers]` is not array

Reply:
Fixed. I got ur point 
thanks
![fixed](https://user-images.githubusercontent.com/17727069/202670274-d37fc532-45a6-482d-ac96-85b0bfb69599.png)